starter-code/server.js

```diff
@@ -23,20 +23,29 @@ app.get('/new', function(request, response) {
 
 app.get('/articles', function(request, response) {
   // REVIEW: This query will join the data together from our tables and send it back to the client.
-  // TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
-  client.query(``)
+  // TO/DO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
+  client.query(`
+    SELECT *
+    FROM articles
+    FULL OUTER JOIN authors
```

Comment:
Unless you're specifically handling articles without authors or authors without articles (you're not), this should be an `INNER JOIN`. You almost always want `INNER JOIN`.